doc-content/enterprise-only/openshift/imagestreams-file-install-amq-proc.adoc

```diff
@@ -0,0 +1,38 @@
+[id='imagestreams-file-install-amq-proc']
+= Ensuring the availability of AMQ scaledown controller image streams for a high-availability deployment
+
+If you want to deploy a {PRODUCT} high-availability authoring environment, you must ensure that image streams for AMQ scaledown controller are available.
+
+.Prerequisites
+* You ensured the availability of image streams and the image registry for {PRODUCT} according to instructions in <<imagestreams-file-install-proc>>.
+
+.Procedure
+
+If you want to deploy a high-availability {CENTRAL} (this functionality is a technology preview), complete the following additional steps:
+
+. Verify if the AMQ scaledown controller image stream is present. Enter the following command:
++
+[subs="attributes,verbatim,macros"]
+----
+$ oc get imagestreamtag -n openshift | grep amq-broker-72-scaledown-controller-openshift
+----
++
+If the output of the command is not empty, the required image stream is available in the `openshift` namespace and no further action is required.
++
+. If the output of the commands is empty, complete the following steps:
+.. Download the following file: https://raw.githubusercontent.com/jboss-container-images/jboss-amq-7-broker-openshift-image/amq-broker-72/amq-broker-7-scaledown-controller-image-streams.yaml
```

Comment:
Suggest one of the following (best to avoid raw URL links, especially when they're long):

```
.. Download the https://raw.githubusercontent.com/jboss-container-images/jboss-amq-7-broker-openshift-image/amq-broker-72/amq-broker-7-scaledown-controller-image-streams.yaml[`amq-broker-7-scaledown-controller-image-streams.yaml`] file.
```

```
.. Download the https://raw.githubusercontent.com/jboss-container-images/jboss-amq-7-broker-openshift-image/amq-broker-72/amq-broker-7-scaledown-controller-image-streams.yaml[AMQ scaledown controller image stream] YAML file.
```